terror_error.go

```diff
@@ -128,10 +118,10 @@ func (e *Error) Error() string {
 	if len(describe) == 0 {
 		describe = ErrCodeText(strconv.Itoa(int(e.code)))
 	}
-	return fmt.Sprintf("[%s] %s", e.RFCCode(), e.getMsg())
+	return fmt.Sprintf("[%s]%s", e.RFCCode(), e.GetMsg())
```

Comment:
how about check `e == nil` in the first line

I ob a stranger question....

we often log some nil terror like this:

![image](https://user-images.githubusercontent.com/528332/91845234-60640b80-ec8b-11ea-9614-e2f5386574cc.png)

and fmt will meet panic and self recovery...

![image](https://user-images.githubusercontent.com/528332/91845168-4a564b00-ec8b-11ea-9f00-c1bcf691f35c.png)

maybe better check nil and return "nil" string?